src/gui/optionsdialog.ui

```diff
@@ -130,7 +130,7 @@
             <item>
              <widget class="QGroupBox" name="UISettingsBox">
               <property name="title">
-               <string>Interface</string>
+               <string>Interface (requires restart)</string>
```

Comment:
@jagannatharjun:  Would it be possible to give the `(requires restart)` text the same *italicized* styling as the removed lines, here? 

If you can do so, I think it would be better for stylistic consistency (within the Options GUI).

Reply:
The problem is QGroupbox's title doesn't support richtext, it can be done with a "hack", but I don't think that will be approved.

Reply:
@jagannatharjun: 

> QGroupbox's title doesn't support richtext 

I suspected that might be the case. 

Would it be possible to simply move the original `(requires restart)` italicized QLabel code, to the same UI row as the groupbox title?

Reply:
@jagannatharjun: 

> QGroupbox's title doesn't support richtext 

I suspected that might be the case.  But it looks to me that the `(requires restart)` text was a separate UI label to begin with. 

That being the case, why not simply move the original `(requires restart)` italicized QLabel, to the same UI row as the groupbox title?

Reply:
afaik, group box title's doesn't have its own row. We have to place the label on top of the title without a layout so it'll will be hard to manage on window resize and other events.

Reply:
Maybe place label "Changing Interface settings requires restart." inside group box itself (as first line)?

Reply:
I totally agree with you. @chocobo1 opinion?

Reply:
>it can be done. @Chocobo1 opinion?

No strong opinion from me..

>Should it be italicized or bold or both?

IMO italicized but not bold.

Reply:
![image](https://user-images.githubusercontent.com/34717789/72345293-58a22780-36f9-11ea-919d-768f57943444.png)